server/paymentProviders/stripe/creditcard.js

```diff
@@ -54,6 +54,12 @@ export default {
      * @param {*} hostStripeAccount
      */
     const getOrCreateCustomerIdForHost = (hostStripeAccount) => {
+      // Customers pre-migration will have their stripe user connected
+      // to the platform stripe account, not to the host's stripe
+      // account. Since payment methods had no name before that
+      // migration, we're using it to test for pre-migration users;
+      if (!paymentMethod.name) return paymentMethod.custommerId;
```

Comment:
`customerId` is misspelled. Worth adding a test for this use case?

Reply:
Thanks for the catch! Yeah, certainly worth adding some tests! I'm currently fixing the ones that broke with this small change.